src/Flex.php

```diff
@@ -587,6 +587,20 @@ private function fetchRecipes(): array
             $name = $package->getNames()[0];
             $job = $operation->getJobType();
 
+            if (!empty($manifests[$name]['manifest']['conflict'])) {
+                $lockedRepository = $this->composer->getLocker()->getLockedRepository();
+
+                foreach ($manifests[$name]['manifest']['conflict'] as $conflictingPackageName => $constraint) {
+                    if ($conflictingPackage = $lockedRepository->findPackage($conflictingPackageName, $constraint)) {
+                        if ($this->io->isVerbose()) {
+                            $this->io->writeError(sprintf('  - Skipping recipe for %s: it conflicts with %s %s.', $name, $conflictingPackageName, $constraint));
```

Comment:
What does the latest version of the message look like? I'm not sure exactly what each variable holds.

Also, does this only show in verbose mode? Does that mean it will not sure unless the user adds `-v`?

Reply:
Ha! Ok, I feel the opposite :p. Well, sort of. Even if you assume that you don't know anything about recipes, a user expects that when they install a package, it will work just like if they follow the docs. In the case of Messenger, it means that the latest Messenger is being installed (so, ideally the user is following the latest messenger docs), but they do NO get the recipe that the docs expect. I don't see a lot of harm in showing it always - it's not a message that they will see over and over again.

I'd also like to add:
> If you want to install the recipe, fix the conflicting package(s) and run composer sync-recipes

Reply:
One wouldn't expect that in the use case that triggered the PR: if you install messenger with framework bundle, you *do not* expect it to be configured (because that's a feature of fwb >=4.1, which you decide not to use.)

Fixing a dep to get a recipe is really the reverse way of thinking: you first require deps, then you configure them, not the opposite.

Reply:
Ok, good for me. I forgot that, for normal users, it's not possible to get a "mix" of symfony/* versions anyways, due to the locking on a specific minor thanks to Flex.